New Import/Export Instructions with TOC update. (#1464)
MicrosoftDocs/SCCMdocs: sccm/apps/deploy-use/copy-applications.md

> ---

Reviewer: I suggest not including this new article, as it's too short. Instead, include the intro paragraph (suggested rewording below) and the process as a new H3 section (`###`) in this existing article: [Manage applications](https://docs.microsoft.com/en-us/sccm/apps/deploy-use/management-tasks-applications#manage-applications). That article talks about other actions in a table format. I'm not a fan of tables for this purpose, so if you also want to break up that table so that each row is a separate H3, that's great. 😄 

Suggestion rewording for intro blurb:

> Copy, or duplicate, Configuration Manager applications to create a new one. This action is useful to test something or when you need to create a similar application.

Author: Since I had made this change in master before (poor newbie mistake on my part), I reforked the actual current master, branched, and then opened #1584 for this one.

title: Copy applications
titleSuffix: Configuration Manager
description: Copy applications to create duplicates in Configuration Manager
ms.date: 05/06/2019
ms.prod: configuration-manager
ms.technology: configmgr-app
ms.topic: conceptual

> #ms.assetid: cc230ff4-7056-4339-a0a6-6a44cdbb2857

Reviewer: Uncomment this and use **New-Guid** cmdlet

> author: ChrisKibble

Reviewer: Change this to me, `aczechowski` (lowercase)

#ms.author: aaroncz
#manager: dougeby
#ms.collection: M365-identity-device-management
---

# Copy Applications in Configuration Manager

*Applies to: System Center Configuration Manager (Current Branch)*

> Configuration Manager applications can be copied (duplicated) when could be useful when creating a new application to test something or when a similar application must be created.

Reviewer: Suggested edit for style:

```suggestion
Copy, or duplicate, Configuration Manager applications to create a new application. This action is useful to test something or when you need to create a similar application.
```
